eng/Tools/DiagConfig/MergeAnalyze.ps1

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env pwsh
```

Comment:
Can we please leave this script in the top-level `scripts` folder? This is where maintenance for the repo are located. Note that this another script in there that uses DiagConfig too for instance.